locale/en/blog/release/v8.0.0.md

```diff
@@ -0,0 +1,1216 @@
+---
+date: 2017-05-30T12:00:00.000Z
+version: 8.0.0
+category: release
+title: Node v8.0.0 (Current)
+slug: node-v8-0-0
+layout: blog-post.hbs
+author: James M Snell, Colin Ihrig
+---
+
+The next major release of Node.js brings a range of significant changes and
+additions, far too many for us to cover adequately in a blog post such as this.
+This article contains a summary of the changes and details we feel are the most
+significant and important.
+
+### Say hello to npm version 5.0.0
+
+npm, Inc. recently
+[announced the release](http://blog.npmjs.org/post/161081169345/v500) of version
+5.0.0 of the npm client and we are happy to include this significant new version
+within Node.js 8.0.0.
+
+### Say hello to V8 5.8
+
+Node.js 8.0.0 ships with V8 5.8, a significant update to the JavaScript runtime
+that includes major improvements in performance and developer facing APIs. Most
+significant for Node.js developers is the fact that V8 5.8 is guaranteed to have
+forwards ABI compatibility with V8 5.9 and the upcoming V8 6.0, which will help
+to ensure stability of the Node.js native addon ecosystem. During Node.js 8’s
+lifetime, we plan to move to 5.9 and possibly even 6.0.
+
+The V8 5.8 engine also helps set up a pending transition to the new
+[TurboFan + Ignition](https://v8project.blogspot.com/2017/05/launching-ignition-and-turbofan.html)
+compiler pipeline, which promises to provide significant new performance
+optimizations for all Node.js applications. Although they have existed in
+previous versions of V8, TurboFan and Ignition will be enabled by default for
+the first time in V8 5.9. The new compiler pipeline represents such a
+significant change that the Node.js Core Technical Committee (CTC) chose to
+postpone the Node.js 8 release, originally scheduled for April, in order to
+better accommodate it.
+
+### Say hello to the Node.js API (N-API)
+
+For Node.js developers who use or create native addons, the new experimental
+Node.js API (N-API) is a significant advancement over the existing
+[Native Abstractions for Node.js (`nan`)](https://www.npmjs.com/package/nan)
+that will allow native addons to be compiled once on a system and used across
+multiple versions of Node.js.
+
+By providing a new virtual machine agnostic Application Binary Interface (ABI),
+it becomes possible for native addons to work not only against multiple
+versions of the V8 JavaScript runtime, but Microsoft's Chakra-Core runtime
+as well.
+
+The [N-API](https://nodejs.org/api/n-api.html) is still considered experimental
```

Comment:
Nit: Remove `considered`. (It's either experimental or it's not. Rather than provide clarity, 'considered' just muddies things.)